arch/x86/entry.S

```diff
@@ -225,6 +225,7 @@ END_FUNC(syscall_handler_entry)
 ENTRY(sysenter_handler_entry)
```

Comment:
please squash this whole commit with https://github.com/KernelTestFramework/ktf/pull/290/commits/b7a0639636a5413363e4250f2aa9802dcc91347c

Reply:
this change is already https://github.com/KernelTestFramework/ktf/commit/b7a0639636a5413363e4250f2aa9802dcc91347c ?